src/client/supabase.ts

```diff
@@ -95,13 +95,15 @@ const handleChangeSales = (
 const handleChangeAllowlistRecords = (
   payload: RealtimePostgresChangesPayload<{ [key: string]: any }>,
 ) => {
-  console.log(payload);
+  console.debug(payload);
   switch (payload.eventType) {
     case "INSERT":
-      cache.invalidate([{ typename: "AllowlistRecord" }]);
-      break;
     case "UPDATE":
-      cache.invalidate([{ typename: "AllowlistRecord", id: payload.new.id }]);
+      cache.invalidate([
```

Comment:
do we really need to invalidate the cache on all allowlist records when 1 gets updated?

Reply:
updated. You can validate with running the API locally and using this query while claiming from testnet.hypercerts.org:

```graphql
{
  allowlistRecords(
    where: {user_address: {eq: >YOUR ADDRESS<}, claimed: {eq: false}}
  ) {
    __typename
    count
    data {
    __typename
      claimed
    }
  }
}
```